input/resources/au-erequesting-patientaccess.xml

```diff
@@ -0,0 +1,325 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CapabilityStatement xmlns="http://hl7.org/fhir">
+  <id value="au-erequesting-patientaccess"/>
+  <extension url="http://hl7.org/fhir/StructureDefinition/structuredefinition-fmm">
+    <valueInteger value="0"/>
+  </extension>
+  <url value="http://hl7.org.au/fhir/ereq/CapabilityStatement/au-erequesting-patientaccess"/>
+  <name value="AUeRequestingPatientAccessCapabilityStatement"/>
+  <title value="AU eRequesting Patient Access CapabilityStatement"/>
+  <status value="draft"/>
+  <description value="This CapabilityStatement describes the basic rules for the [AU eRequesting Patient Access actor](ActorDefinition-au-erequesting-actor-patientaccess.html) that is  is a system that accepts diagnostic service requests and makes diagnostic service requests accessible. The complete list of FHIR profiles, RESTful operations, and search parameters supported by AU eRequesting patient access clients are defined in this CapabilityStatement."/>
```

Comment:
update description -  digital interface that allows patients or their representatives to view and manage requests for diagnostic services.

Reply:
fixed